frontend/src/pages/RidePage.tsx

```diff
@@ -6,16 +6,31 @@ import { useObjectVal } from "react-firebase-hooks/database";
 import { db } from "../firebase";
 import { Box, Heading } from "@chakra-ui/react";
 import { latLng, LatLng } from "leaflet";
+import { icon } from "leaflet";
+import startIconImg from "../images/Arrow Circle Up_8.png";
+import endIconImg from "../images/Arrow Circle Down_8.png";
+
+export const defaultMapCenter = latLng([48.46557, -123.314736]);
 
 export default function RidePage() {
   const { rideId } = useParams();
   const [ride] = useObjectVal<Ride>(ref(db, "rides/" + rideId));
-  let center = latLng([48.46557, -123.314736]);
+  let center = defaultMapCenter;
   let startMarker, endMarker;
   if (ride) {
     center = findMidpoint(latLng(ride.start), latLng(ride.end));
-    startMarker = <Marker position={ride.start} />;
-    endMarker = <Marker position={ride.end} />;
+    const startIcon = icon({
+      iconUrl: startIconImg,
+      iconSize: [96, 96],
+      iconAnchor: [48, 92],
+    });
+    const endIcon = icon({
+      iconUrl: endIconImg,
+      iconSize: [96, 96],
```

Comment:
These should probably be defined as constants at the top of the file instead of having "magic numbers" in the code.

Reply:
I agree, just didn't know where to put them. I'd like to put asset constants in a separate place though as they'll get used elsewhere. I'll make this a separate issue.